float64	l_spd
float64	r_spd
float64	l_pos
float64	r_pos